source/_components/ptvsd.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: "PTVSD Debugger"
```

Comment:
I suggest we add `(Visual Studio Code)` behind the name, so people know what it's about.

```suggestion
title: "PTVSD Debugger (Visual Studio Code)"
```